btcutil/psbt/psbt.go

```diff
@@ -37,6 +37,9 @@ const MaxPsbtValueLength = 4000000
 // deserialize from the wire. Anything more will return ErrInvalidKeyData.
 const MaxPsbtKeyLength = 10000
 
+// MaxPsbtKeyValue is the maximum value of a key type in a PSBT.
+const MaxPsbtKeyValue = 0x02000000
```

Comment:
Just to clarify, I think the BIP doesn't specify a maximum value of a key type in a PSBT. Bitcoin Core uses this maximum value for every CompactSize occurrence.

Reply:
Correct. Without this check the value mentioned in the issue wouldn't have caused the test to fail. So I assumed we want to mimic `bitcoind`'s behavior here.
Do you think that warrants a sentence in the constant's comment?

Reply:
Yes, it would be great.

Reply:
Done.